state/state.go

```diff
@@ -47,8 +51,11 @@ type State struct {
 	// AppHash is updated after Commit
 	AppHash []byte
 
+	// XXX: do we need this json tag ?
```

Comment:
We need it so that go-data doesn't serialize it, yeah.

Reply:
But this doesn't get JSON serialized, just go-wire. Does this tag work for non-json go-wire?

Reply:
yes, see reflect.go line 41